test/unit/Cardano/Wallet/Primitive/ModelSpec.hs

```diff
@@ -85,6 +96,14 @@ spec = do
         it "applyBlock matches the basic model from the specification"
             (checkCoverage prop_applyBlockBasic)
 
+    describe "Wallet transactions over test data" $ do
+        -- it "Outgoing transactions have inputs that belonged to the utxo"
+        --    (property prop_applyBlockTxHistoryInputs)
+        it "Incoming transactions have output addresses that belong to the wallet"
+            (property prop_applyBlockTxHistoryIncoming)
+        it "Any transction involving our addresses must be incoming"
+            (property prop_applyBlockTxHistoryOurs)
```

Comment:
I don't get this second property :thinking: ? What about outgoing transactions with change?